src/components/step/ConnectDeviceAccountStepContainer/ConnectDeviceAccountStepContainer.tsx

```diff
@@ -2,13 +2,34 @@ import React, { useState, useEffect } from "react";
 import MyDataHelps, { StepConfiguration } from "@careevolution/mydatahelps-js";
 import ConnectDeviceAccountStep from "../ConnectDeviceAccountStep";
 
-export default function () {
+export interface ConnectDeviceAccountStepContainerProps {
+    deviceType?: string;
+    providerID?: number;
+}
+
+export default function (props: ConnectDeviceAccountStepContainerProps) {
     const [title, setTitle] = useState<string>();
     const [text, setText] = useState<string>();
     const [styles, setStyles] = useState<any>({});
-    const [deviceType, setDeviceType] = useState<string>("");
-    const [providerID, setProviderID] = useState<number>();
-    const [connected, setConnected] = useState<boolean>();
+    const [deviceType, setDeviceType] = useState<string>(
+        props.deviceType ?? ""
+    );
+    const [providerID, setProviderID] = useState<number>(props.providerID ?? 0);
+
+    function onConnect() {
+        MyDataHelps.connectExternalAccount(providerID, {
+            openNewWindow: true,
+        }).then(function () {
+            MyDataHelps.completeStep(providerID);
+        });
+    }
+
+    function isProd() {
+        return (
+            window.location.host.endsWith(".com") ||
```

Comment:
This isn't the right logic; you need to check MyDataHelps.baseUrl like we do here: https://github.com/CareEvolution/MyDataHelpsUI/blob/main/src/components/container/ConnectFitbit/ConnectFitbit.tsx#L18

Note that MyDataHelps.baseUrl won't be populated until you have a token - so you should probably invoke getExternalAccounts first

Reply:
I'm including both https://mdhorg.ce.dev and https://mydatahelps.dev, since both of them are in use.